src/tests/commonTests.ts

```diff
@@ -222,30 +222,41 @@ interface FocusableOptions {
 /**
  * Helper for asserting that a component is focusable
  *
+ * Note that this helper should be used within a describe block
+ *
+ * describe("is focusable", () => {
+ *    focusable(`calcite-input-number`, {shadowFocusTargetSelector: "input" })
```

Comment:
Nitpick: can you make sure there's a space between the curly braces for the `focusable` options?